### ISSUE TITLE
Table 'phplistdb.phplist_i18n' missing when using a docker image tag

### ISSUE THREAD

**Issue by sebastian13** [phpList/phplist-docker#15]
Hi!

I'm experiencing the following error when using a specific docker image tag in `phplist.yml`. Everything is working fine when using the _latest_ tag. 

`phpList - Database error 1146 while doing query  Table 'phplistdb.phplist_i18n' doesn't exist`

## Steps to reproduce

- `git clone https://github.com/phpList/phplist-docker.git .`
- run `cp .env-dist .env`
- In `phplist.yml` change line 24 `image: phplist/phplist:latest` to `image: phplist/phplist:3.6.2`
- run `./start-phplist.sh`
- docker logs will show the following
  ```
  ➜  phplisttest git:(master) ✗ docker logs phplisttest_phplist_1 | head -n 20
  Initialising phpList, Please wait
  Waiting for the Database to be available - 1/10
  phpList - phpList version 3.6.2 (c) 2000-2021 phpList Ltd, https://www.phplist.com
  Error: initialise does not process commandline
  phpList - phpList version 3.6.2 (c) 2000-2021 phpList Ltd, https://www.phplist.com
  A new version of phpList is available: phpList 3.6.4 Download the new versionor use the phpList Updater
  phpList - ar
  phpList - Initialising language ar
  phpList - Database error 1146 while doing query  Table 'phplistdb.phplist_i18n' doesn't exist
  phpList - Sql error replace into phplist_i18n (lan,original,translation) values("ar","import is not available","استيراد غير متوفر")
  phpList - Database error 1146 while doing query  Table 'phplistdb.phplist_i18n' doesn't exist
  phpList - Sql error replace into phplist_i18n (lan,original,translation) values("ar","statistics","إحصائيات")
  ...
  ```

**Comment by michield** [comment]
The docker image does not include the database, so something must have gone wrong when initialising your system.

You can re-create missing tables, by manually running initialise in the UI

https://demo.phplist.org/lists/admin/?page=initialise

**Comment by sebastian13** [comment]
Well, but isn't that also the case for the _latest_ image? The database is always created from scratch when starting via docker-compose, but the command `phplist -pinitialise` seems to work on _latest_, but not on _3.6.2._. I did try this on different machines, so I would be surprised if the issue is related to "my system".

I did just try your UI fix. `http://localhost:8000/lists/admin/?page=initialise` fixes the issue indeed. How is this different from the `/usr/bin/phplist -pinitialise` command in the docker-entrypoint.sh?
